static/src/javascripts/es6/projects/common/modules/crosswords/anagram-helper/clue-input.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+export default class ClueInput extends React.Component {
+    componentDidMount () {
+        React.findDOMNode(this).focus();
+    }
+
+    componentWillReceiveProps (props) {
+        // focus on reset
+        if (props.value === '') {
+            const node = React.findDOMNode(this);
+            setTimeout(node.focus.bind(node), 0);
```

Comment:
`setTimeout(, 0)` is relatively bad for [performances](https://developer.mozilla.org/en-US/docs/Web/API/WindowTimers/setTimeout#Notes), it hogs the CPU, better use 4 or 10